go.mod

```diff
@@ -56,6 +55,7 @@ require (
 	k8s.io/kubectl v0.17.4
 	sigs.k8s.io/controller-runtime v0.5.2
 	sigs.k8s.io/controller-tools v0.2.8
+	sigs.k8s.io/yaml v1.1.0
```

Comment:
Note: this needs to be bumped to [v1.2.0](https://github.com/kubernetes-sigs/yaml/releases/tag/v1.2.0) when we bump to k8s 1.18.

Reply:
Added a note as well.